gremlin-python/pom.xml

```diff
@@ -332,6 +332,15 @@ limitations under the License.
                                               failonerror="true">
                                             <arg line="-m pip install pip&gt;=19.3.1,&lt;20.0.0"/>
                                         </exec>
```

Comment:
?

Reply:
I pulled and rebased to the current 3.4, so these are the new changes.